torchtune/utils/metric_logging.py

```diff
@@ -176,14 +176,15 @@ def __init__(
         _, self.rank = get_world_size_and_rank()
 
         if self.rank == 0:
-            self._wandb.init(
+            run = self._wandb.init(
                 project=project,
                 entity=entity,
                 group=group,
                 reinit=True,
                 resume="allow",
                 **kwargs,
             )
+            run._label(repo="torchtune")
```

Comment:
Can you explain this a little more? What does the `repo` kwarg enable?

Reply:
We get telemetry on the repo usage. We already do this by grabbing the git repo used to train but if people rename it or fork it can mess up the identification. We can share this data with you guys to see adoption.

Reply:
In practice, it is just one extra key in: https://wandb.ai/capecape/torchtune/runs/ynahtfyw/files/config.yaml
the key `9`

Reply:
Makes sense, thanks for the explanation! (We would love to see adoption numbers soon :) )